HAB5/hab5.tex

```diff
@@ -0,0 +1,404 @@
+
+The primary payload utilized is the WUAP2 payload. 
+This is to build on the previous WUAP payload while adding a NDVI system.
+Use of NDVI is to be investigated on the ground and considerable attention paid to how NDVI results may be linked to visual spectrum analysis of vegetation. 
+While specific to the payload mission selected, exploring the linking of different realms of sensor information is a pursuit which will further aid SPEX. 
+An example could be the linking of IMU attitude data to images at the corresponding time, which in turn may allow for correction of apparent areas due to the angle of the camera with respect to the horizon. 
+This in turn prepares SPEX for projects requiring this correction be done on-flight, and for other projects such as horizon detection which could ``train'' on-flight by comparing analyzed data to known IMU readings. 
+To this end the synchronization of clocks may be pursued. 
+Another consistent challenge has been the processing of information on a stream, without information being skipped or resampled. 
+Raw and processed information should be saved for later evaluation and iteration of systems.
+
+Significant departures from prior procedures in this proposition include the rigorous testing of components and assembled hardware before flight, training/tuning of algorithms where needed and the inclusion of metrics to detect components going offline, development and propagation of errors. 
+As SPEX moves to consider CV modules, it is expected that the development of training knowledge in pre-flight testing will become crucial. 
+Among the tests are those for: camera alignment; camera stability; spectral response/quantum efficiency; software processing of simulations; payload NDVI/visual functioning while on ground; an electronics functional test with specific focus on power systems and APRS reliability; electronics environmental testing; integrated system testing.
+Time-stamping of information received by different processes may be considered. 
+It is also desired that metrics be included for post-flight analysis.
+
+Among the physical requirements for the payload, it is required that the HAB vehicle and payload system be better capable of protecting from environmental factors such as humidity, winds and temperature. Specific to this payload, the alignment of cameras is a question needing further exploration. An angle and separation optimal for the combination of data acquisition and securedness is required. Vibration or displacement during flights can significantly impair or corrupt all information gathered. Fixing this hurdle is critical for future projects which may look to infer distance data with a combination of two cameras or a radar altimeter setup. While the project doesn't expect that these will all be met as results, it is firmly held that the steps taken to implement these goals as best as possible will transformatively improve SPEX's advanced payload capabilities.
```

Comment:
Do you have maximum/minimum/optimal ranges for internal humidity and temperature inside the HAB?

What kind of risks do winds present and what mitigation strategies could be used? Is the goal to stabilize the camera platform?

Reply:
Range will be written up based on information for all electronic components.
Stabilization is an important one, as is having batteries not break apart.